tsl/src/continuous_aggs/insert.c

```diff
@@ -155,6 +155,14 @@ cache_inval_entry_init(ContinuousAggsCacheInvalEntry *cache_entry, int32 hyperta
 	/* NOTE: we can remove the id=>relid scan, if it becomes an issue, by getting the
 	 * hypertable_relid directly from the Chunk*/
 	Hypertable *ht = ts_hypertable_cache_get_entry_by_id(ht_cache, hypertable_id);
+	if (ht == NULL)
+	{
+		ereport(ERROR,
+				(errcode(ERRCODE_UNDEFINED_OBJECT),
+				 errmsg("unable to get relid for hypertable %d in cache_inval_entry_init",
```

Comment:
```suggestion
				 errmsg("unable to get relid for hypertable %d from cache",
```
Nit... IMHO is a better message for the user, but is up to you!

Reply:
I think the error is actually misleading cause it's an invalid hypertable but the error message seems to suggest that the hypertable could not be found in cache